recipes/flatbuffers/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# C++11 finagling
+if [ "$(uname)" == "Darwin" ]; then
+  # for Mac OSX
+  export CC=clang
+  export CXX=clang++
+  export MACOSX_VERSION_MIN="10.7"
+  export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LDFLAGS="${LDFLAGS} -stdlib=libc++ -std=c++11"
+  export LINKFLAGS="${LDFLAGS}"
+  export MACOSX_DEPLOYMENT_TARGET=10.7
+fi
+
+cmake -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX -DFLATBUFFERS_BUILD_TESTS=OFF .
+make
+make install
```

Comment:
What @msarahan means is we like to run the code's test suite during the build process. Generally, we don't expect to find bugs in the code (though sometimes that happens), but in our build by doing this. It makes sure that we have exercised the package fairly well and can be relatively confident that the binary we ship works in most cases. If we can add a `make check`, `ctest`, or similar step here, that would be great.